build/target-repository/.github/workflows/publish_docker_images.yaml

```diff
@@ -33,7 +33,7 @@ jobs:
               uses: peter-evans/dockerhub-description@v2
               with:
                   username: ${{ secrets.DOCKER_USERNAME }}
-                  password: ${{ secrets.DOCKER_PASSWORD }}
+                  password: ${{ secrets.DOCKERHUB_PASSWORD }}
```

Comment:
It seems this has no effect, as both are aliases:
https://github.com/peter-evans/dockerhub-description/pull/12/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R22

Also it suggest there are 2 passwords needed. There should be just one, to avoid WTF in the future